src/components/sound-picker/sound-picker.tsx

```diff
@@ -7,17 +7,36 @@ import LabelsIcon from "../../assets/icons/sell_black_48dp.svg";
 import "./sound-picker.scss";
 
 export interface ISoundPickerProps {
-  selectedSound: SoundName
+  selectedSound: SoundName;
+  drawWaveLabels: boolean;
   handleSoundChange?: (event: ChangeEvent<HTMLSelectElement>) => void;
   onRecordingCompleted?: (audioBuffer: AudioBuffer) => void;
+  handleDrawWaveLabelChange?: () => void;
 }
 
+export const isPureTone = (soundName: string) => {
+  switch (soundName) {
+    case "middle-c": return true; break;
+    case "c2": return true; break;
+    default: return false;
+  }
+};
+
+export const pureToneFrequencyFromSoundName = (soundName: string) => {
+  switch (soundName) {
+    case "middle-c": return 261.65; break;
+    case "c2": return 65.41; break;
```

Comment:
Same comment as above on `break`. Furthermore, this seems like a maintenance headache. If sounds are added or changed then multiple places in the code need to be revisited. It seems like this could be put into a table or map so that adding/changing/removing sounds could be data-driven rather than requiring additional code changes.

Reply:
I agree that a refactoring, to centralize the sound-choice related code is a good thing to do. However, after discussion with Leslie, she said that it should be deferred until we next start major work on the SoundWaves application.